moped-editor/src/views/projects/projectView/ProjectComponents/ComponentListItem.js

```diff
@@ -60,7 +70,9 @@ export default function ComponentListItem({
           primary={primary}
           secondary={
             <>
-              <>{secondary}</>
+              <span className={!isExpanded && classes.truncatedText}>
+                {secondary}
+              </span>
```

Comment:
when the ListItem is clicked on, (aka expanded), the truncated style is removed so the entire description is visible.